ReleaseNotes.md

```diff
@@ -13,6 +13,104 @@ Release Notes
 
 </div>
 
+<div id="24.12">
+ 
+## OpenRocket 24.12 beta 1
+
+Notable features and fixes in *bold*.
+
+### RELEASE HIGHLIGHTS
+* **Overhauled Simulation tab UI**:  This is the beginning of a multi-release effort to make it easier to manage simulations and interpret their results. Please let us know what you think so far.
+* **Multi-level wind input**
+* **Plot and export Component Analysis parameter sweeps**: See the new Plot/Export tab in Component Analysis and give it a try.
+* **Export fin shapes to SVG file**: For use with laser cutters and import into other tools.
+* **Project documentation moved to Sphinx**: See it at https://openrockt.readthedocs.io/.
+
+### Simulation
+As mentioned above, a lot of work went into the Simulation tab for this release, and there is a lot more to come.
+
+#### Flight simulation
+* **Overhauled Sim table GUI** (fixes 2456)
+* **Multi-level wind input** (fixes 922, 2060, 2558)
+* Sync wind speed, deviation, and turbulence widgets together in sim settings (fixes 2388)
+* Add South/West units for latitude and longitude of launch site (fixes 2178)
+* Improved accuracy of thrust calculation
+* Abort sim when recovery deployment occurs under thrust
+* Allow sustainer to tumble before apogee; if under thrust, abort sim
+* Fire outdated sim on stage rename (fixes 2532)
+* Added "aborted" status mark to simulations
+* Don't show vertical acceleration as negative before liftoff
+
```

Comment:
Also add "Clamp compressibility correction factor to avoid singularity (#2570)"?